x-pack/plugins/apm/server/routes/services/get_services/get_service_transaction_stats.ts

```diff
@@ -90,28 +92,35 @@ export async function getServiceTransactionStats({
           },
         },
         aggs: {
-          services: {
-            terms: {
-              field: SERVICE_NAME,
-              size: maxNumServices,
+          sample: {
+            random_sampler: {
+              probability,
```

Comment:
aye, any idea what that should/can be? can it be something like "apm-app"?

Reply:
@dgieselaar just confirming that for anything that is a visualization, the progressive fetching is seeded for a user's session :)

Reply:
Do you mean that this is handled in Elasticsearch?

Reply:
@dgieselaar no, it is not. Elasticsearch doesn't know about kibana user's sessions. 

If the random_sampler is used for visualizations, it should be seeded. Otherwise a different subset of the data is used on every search call, which would be jarring as the visualization will subtly jump around.

Reply:
Dang, too bad, because I forgot about it. I'll create a follow-up issue. I don't think this is required for an experimental feature though.

Reply:
https://github.com/elastic/kibana/issues/130784